aion/web/controllers/question_controller.ex

```diff
@@ -17,12 +19,16 @@ defmodule Aion.QuestionController do
       question_changeset = Question.changeset(%Question{}, Map.put(question_params, "belongs_to", question_subject))
       case Repo.insert(question_changeset) do
         {:ok, question} ->
-          answer_changeset = Answer.changeset(%Answer{}, %{"content" => answers_content, "belongs_to" => question})
-          case Repo.insert(answer_changeset) do
-            {:ok, answer} ->
-              question
-            {:error, error_answer_changeset} ->
-              Repo.rollback(error_answer_changeset)
+          insert_asnwers =
+            answers_content
```

Comment:
Props for newline at the beginning of the pipe chain, was not sure but checked that in [Elixir style guide](https://github.com/christopheradams/elixir_style_guide#pipe-operator) and now I know how it should be done properly.